client/taskenv/env.go

```diff
@@ -104,11 +110,12 @@ const (
 
 // The node values that can be interpreted.
 const (
-	nodeIdKey     = "node.unique.id"
-	nodeDcKey     = "node.datacenter"
-	nodeRegionKey = "node.region"
-	nodeNameKey   = "node.unique.name"
-	nodeClassKey  = "node.class"
+	nodeIdKey            = "node.unique.id"
+	nodeDcKey            = "node.datacenter"
+	nodeRegionKey        = "node.region"
+	nodeNameKey          = "node.unique.name"
+	nodeClassKey         = "node.class"
+	nodeComputedClassKey = "node.computedClass"
```

Comment:
I missed this before, but I think this must have gotten left in from some other change you're working on, as it's not related to environment variables. (I'm also not sure why we'd want to expose this internal value but let's talk about that when you open that PR.)

Reply:
Yeah this is a left over from playing around with something else. Sorry about that - removed